src/main/scala/org/biodatageeks/sequila/utils/InternalParams.scala

```diff
@@ -54,7 +54,6 @@ object InternalParams {
   final val maxGap = "spark.biodatageeks.rangejoin.maxGap"
   final val minOverlap = "spark.biodatageeks.rangejoin.minOverlap"
   final val intervalHolderClass = "spark.biodatageeks.rangejoin.intervalHolderClass"
-
-
+  final val domains = "spark.biodatageeks.rangejoin.domains"
```

Comment:
Rather name it as spark.biodatageeks.rangejoin.iitii.domainsNum

Reply:
Add documentation to : https://biodatageeks.github.io/sequila/docs/configuration/join/

Reply:
As well section on available algos in a tabular form:
https://biodatageeks.github.io/sequila/docs/algorithms/join/